Modules/mathmodule.c

```diff
@@ -2496,6 +2496,146 @@ math_isclose_impl(PyObject *module, double a, double b, double rel_tol,
 }
 
 
+/*[clinic input]
+math.comb
+
+    a: object
+
+    b: object
```

Comment:
I would advice to mark these as positional only arguments (use the "/" of the argument clinic).

Reply:
Done. Thanks